spec/music_album_spec.rb

```diff
@@ -0,0 +1,23 @@
+require_relative '../music_album'
+
+describe MusicAlbum do
```

Comment:
As per the project requirements, `can_to_archive` method in the music_album class should also be tested. Kindly follow up on this requirement.